share/spice/open_nic/open_nic.css

```diff
@@ -0,0 +1,64 @@
+/*
+    Document   : zeroclickinfo-spice/share/spice/open_nic/open_nic.css
+    Created on : 2015-08-17 04:49 PM
+    Author     : cylgom
+    Description:
+    Arranges the list to look more like a table
+*/
+
+root {
```

Comment:
whoops, please remove this, or namespace it with `.zci--open_nice`as It will affect every element on the page!